_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -117,45 +123,242 @@ This `button` element has the text "search" rendered as an magnifying glass icon
+```
+
+#### Passed Example 11
+
+This example shows that the [label in name algorithm][] handles many kinds of whitespace.
+
+```html
+<a aria-label="compose email" href="#">compose &nbsp;&nbsp;<br> email</a>
+```
+
+#### Passed Example 12
+
+This example passes the rule because "YYYY-MM-DD" is in brackets.  Text in brackets is removed by the [label in name algorithm][], because its not normally spoken by speech-input users.  
+
+```html
+<button aria-label="Search by date">Search by date (YYYY-MM-DD)</button>
+```
+
+#### Passed Example 13
+
+The passes for two reasons: 1) because the ellipsis ("…") is [non-text content][], and 2) because the ellipsis is neither a letter nor a digit and so is filtered out by the [label in name algorithm][].
+
+```html
+<button aria-label="Next">Next…</button>
+```
+
+#### Passed Example 14
+
+This passes because the [label in name algorithm][] effectively ignores all punctuation and emoji, in both the visible inner text and the accessible name, as long as they don't break up words.
+
+```html
+<button aria-label="💡 Submit 💡">>>> ** Submit ** <<<</button>
+```
+
+#### Passed Example 15
+
+The "X" is non-text content.
+
+```html
+<button aria-label="Close">X</button>
+```
+
+
+
 ### Failed
 
 #### Failed Example 1
 
-This link has [visible][] text that is different from the [accessible name][].
+This link has [visible inner text][] that is very different from the [accessible name][].
 
 ```html
 <a href="https://act-rules.github.io/" aria-label="WCAG">ACT rules</a>
 ```
 
 #### Failed Example 2
 
-This button has [visible][] text that is only partially contained within the [accessible name][].
+This button has [visible inner text][] that is only partially contained within the [accessible name][].
 
 ```html
 <button aria-label="the full">The full label</button>
 ```
 
-#### Failed Example 3
+#### Failed Example 3 
+
+This button has [visible inner text][] that is fully contained within the [accessible name][] when viewed as a character-by-character substring.  But that does not satisfy our [label in name algorithm][], which works on full words.  So this fails the rule.
+
+```html
+<a href="#" aria-label="Discover Italy">Discover It</a>
+```
+
+#### Failed Example 4 
+
+This link's [accessible name][] contains two tokens (according to the[label in name algorithm][]) and the [visible inner text][] contains one token.  So it fails the rule.
+
+```html
+<a aria-label="just ice" href="#">justice</a>
+```
+
+#### Failed Example 5 
+
+This link has an [accessible name][] which contains a hyphen.  The [label in name algorithm][] breaks up words on hyphens.  So it turns "non-standard" into two tokens: "non" and "standard".  So this fails the rule.
+
+```html
+<a href="#" aria-label="non-standard">nonstandard</a>
+```
+
+#### Failed Example 6 
+
+The rule has no special handling for acronyms or initialisms.
+
+```html
+<a aria-label="WAVE" href="#">W A V E</a>
```

Comment:
We should take a name that isn't also an Accessibility vendor.
Maybe simply "WCAG"?

Reply:
Done.